src/ceph-detect-init/Makefile.am

```diff
@@ -53,7 +53,7 @@ EXTRA_DIST += \
 	ceph-detect-init/tests/test_all.py \
 	ceph-detect-init/tox.ini
 
-export CEPH_DETECT_INIT_VIRTUALENV = ${CEPH_BUILD_VIRTUALENV}/ceph-detect-init-virtualenv
+export CEPH_DETECT_INIT_VIRTUALENV = ${CEPH_BUILD_VIRTUALENV}ceph-detect-init-virtualenv
```

Comment:
does this mean that the `CEPH_BUILD_VIRTUALENV` env var needs to end in a trailing slash to work properly?